core/trino-main/src/main/java/io/trino/operator/output/PartitionedOutputOperator.java

```diff
@@ -356,34 +357,37 @@ public Page getOutput()
         return null;
     }
 
-    public static class PartitionedOutputInfo
-            implements Mergeable<PartitionedOutputInfo>, OperatorInfo
+    /**
+     * Keeps statistics about output pages produced by the partitioner + updates the stats in the operatorContext.
```

Comment:
I don't think th comment is valid anymore as it's merely a supplier now

Reply:
that's true. I dropped the commit.